### DISCUSSION TITLE
GPU at 87C on most games (Zephyrus M16)

### DISCUSSION THREAD

**Discussion by MinhNguyen4958** [seerge/g-helper#1002] [Q&A] [answered]
The default value for GPU's Temperature Target is 87 in G-Helper. I notice that the GPU temp is trying to reach that target no matter what games. There are no performance hits tho, so that's a relief, but I'm afraid for my laptop's longevity. Is it normal to game in such temperature for long time?

**Comment by seerge** [comment, marked as answer]
87C is default temp limit from Asus. Temp is not "trying to reach" it :) it just goes up when you use your device, and this is normal :)

**Reply by bmagnien** [reply]
@seerge is there a way to override this? i notice the gpu starts downclocking around 85/86c but not sure if that's something g-helper is able to overcome. not sure why the gpu temp cap is set so 'low' at 87 by default...

**Reply by seerge** [reply]
No, it's hardcoded (by asus) in the range G-Helper already lets you set it to

**Reply by bmagnien** [reply]
got it. i imagine it's vbios controlled. i've already flashed the scar 16 vbios to the g14's 4090 - but the temp limit is the same there. thanks for the prompt reply :)
